lms/djangoapps/grades/tests/test_new.py

```diff
@@ -381,3 +381,141 @@ def test_max_score_does_not_change_results(self, alterations):
+        self.problem = ItemFactory.create(
+            parent=self.vertical,
+            category="problem",
+            display_name="Test Problem 1",
+            data=problem_xml
+        )
+        self.problem_2 = ItemFactory.create(
+            parent=self.vertical_2,
+            category="problem",
+            display_name="Test Problem 2",
+            data=problem_xml
+        )
+        self.problem_3 = ItemFactory.create(
+            parent=self.vertical_3,
+            category="problem",
+            display_name="Test Problem 3",
+            data=problem_xml
+        )
+        self.request = get_request_for_user(UserFactory())
+        self.client.login(username=self.request.user.username, password="test")
+        self.course_structure = get_course_blocks(self.request.user, self.course.location)
+        self.subsection_grade_factory = SubsectionGradeFactory(self.request.user, self.course, self.course_structure)
+        CourseEnrollment.enroll(self.request.user, self.course.id)
+
+    def _create_course_grade_and_check_logging(
+            self,
+            factory,
+            log_mock,
+            read_only,
+            subsections_read,
+            subsections_created,
+            blocks_accessed,
+            total_graded_subsections
+    ):
+        """
+        Creates a course grade and asserts that the associated logging
+        matches the expected totals passed in to the function.
+        """
+        factory.create(self.course)
+        log_statement = u''.join((
+            u"compute_and_update, read_only: {0}, subsections read/created: {1}/{2}, blocks ",
+            u"accessed: {3}, total graded subsections: {4}"
+        )).format(
+            read_only,
+            subsections_read,
+            subsections_created,
+            blocks_accessed,
+            total_graded_subsections,
+        )
+        log_mock.warning.assert_called_with(
+            u"Persistent Grades: CourseGrade.{0}, course: {1}, user: {2}".format(
+                log_statement,
+                unicode(self.course.id),
+                unicode(self.request.user.id),
+            )
+        )
+
+    def test_course_grade_logging(self):
+        grade_factory = CourseGradeFactory(self.request.user)
+        with persistent_grades_feature_flags(
+            global_flag=True,
+            enabled_for_all_courses=False,
+            course_id=self.course.id,
+            enabled_for_course=True
+        ):
+            with patch('lms.djangoapps.grades.new.course_grade.log') as log_mock:
+                # the course grade has not been created, so we expect each grade to be created
+                self._create_course_grade_and_check_logging(
+                    grade_factory,
+                    log_mock,
+                    read_only=False,
+                    subsections_read=0,
+                    subsections_created=3,
+                    blocks_accessed=3,
+                    total_graded_subsections=2)
```

Comment:
Nit: per our convention, closing paren should be on its own line.  And you can then add a trailing comma as well.